packages/paste-core/components/alert/package.json

```diff
@@ -50,11 +50,8 @@
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",
-    "@emotion/core": "^10.0.10",
```

Comment:
These were all added based on the output of the package doctor, and it kind of makes sense when you take into consideration that this package needs to build on its own. Peer dependencies are installed by a host app, but for the sake of building this component, the host app is itself, hence all the peer deps that are needed for the build are listed as dev deps